test/stage2/test.zig

```diff
@@ -1477,4 +1477,58 @@ pub fn addCases(ctx: *TestContext) !void {
             \\}
         , &[_][]const u8{":8:10: error: evaluation exceeded 1000 backwards branches"});
     }
+    {
+        var case = ctx.exe("@setEvalBranchQuota", linux_x64);
+        case.addCompareOutput(
+            \\export fn _start() noreturn {
+            \\    @setEvalBranchQuota(100000);
```

Comment:
Let's avoid wasting time since this test is going to be run countless times. We should have exactly 1 test setting the quota to exactly 1001 branches, and the rest should be using much lower values.